packages/theme-toolkit/src/component-stories-utrecht.tsx

```diff
@@ -1907,15 +1903,241 @@ export const UTRECHT_COMPONENT_STORIES: ComponentStory[] = [
+          <TableRow>
+            <TableHeaderCell scope="col">Column Heading Cell 1</TableHeaderCell>
+            <TableHeaderCell scope="col">Column Heading Cell 2</TableHeaderCell>
+            <TableHeaderCell scope="col">Column Heading Cell 3</TableHeaderCell>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 1</TableHeaderCell>
+            <TableCell>Data Cell 1</TableCell>
+            <TableCell>Data Cell 2</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 2</TableHeaderCell>
+            <TableCell>Data Cell 3</TableCell>
+            <TableCell>Data Cell 4</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 3</TableHeaderCell>
+            <TableCell>Data Cell 5</TableCell>
+            <TableCell>Data Cell 6</TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+    ),
+    detectTokens: {
+      anyOf: [
+        'utrecht.table.footer.color',
+        'utrecht.table.footer.background-color',
+        'utrecht.table.footer.sticky.color',
+        'utrecht.table.footer.sticky.background-color',
+      ],
+    },
+  },
+  {
+    storyId: 'react-utrecht-table--cell',
+    component: 'utrecht-table',
+    name: 'Utrecht Table with cell',
+    render: () => (
+      <Table>
+        <TableCaption>Caption of the Table</TableCaption>
+        <TableHeader>
+          <TableRow>
+            <TableHeaderCell scope="col">Column Heading Cell 1</TableHeaderCell>
+            <TableHeaderCell scope="col">Column Heading Cell 2</TableHeaderCell>
+            <TableHeaderCell scope="col">Column Heading Cell 3</TableHeaderCell>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 1</TableHeaderCell>
+            <TableCell>Data Cell 1</TableCell>
+            <TableCell>Data Cell 2</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 2</TableHeaderCell>
+            <TableCell>Data Cell 3</TableCell>
+            <TableCell>Data Cell 4</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 3</TableHeaderCell>
+            <TableCell>Data Cell 5</TableCell>
+            <TableCell>Data Cell 6</TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+    ),
+    detectTokens: {
+      anyOf: [
+        'utrecht.table.cell.line-height',
+        'utrecht.table.cell.padding-block-end',
+        'utrecht.table.cell.padding-block-start',
+        'utrecht.table.cell.padding-inline-end',
+        'utrecht.table.cell.padding-inline-start',
+        'utrecht.table.cell.selected.color',
+        'utrecht.table.cell.selected.background-color',
+        'utrecht.table.cell.icon.size',
```

Comment:
```suggestion
```

Er is hier geen icon om te kunnen tweaken, deze token heeft z'n eigen story nodig.